help/destinations/catalog/adobe/marketo-engage-person-sync.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Marketo Engage Person Sync
+description: Use the Marketo Engage Person Sync connector to stream updates from a person audience to the corresponding records in your Marketo Engage.
+---
+
+# Marketo Engage Person Sync connection {#marketo-engage-person-sync}
+
```

Comment:
I understand that this is an alpha or beta destination at first, so please add a top-of-page note to call that out. 

Maybe something like:

```
>[!IMPORTANT]
>
>This destination connector is in beta and only available to select customers. To request access, contact your Adobe representative.
```